sdk/formrecognizer/ai-form-recognizer/test/browser/formtrainingclient.spec.ts

```diff
@@ -169,6 +171,7 @@ describe("FormTrainingClient browser only", () => {
 
   it("getModel() returns a model", async function() {
     if (!modelIdToDelete) {
+      // eslint-disable-next-line no-invalid-this
       this.skip();
```

Comment:
We can use the below pattern to avoid the `this.skip()`
```
if (modeIdToDelete) {
     it("getModel() returns a model", async function() {});
}
```

Reply:
I would recommend against it, `this.skip()` nicely shows the tests that are skipped in the report.

If we wrap the tests with "if" conditions, I fear there is a possibility that those tests would never get executed and would go unnoticed.

Reply:
makes sense

Reply:
So this test rely on the id of a model that is created in a previous test.  If that test failed resulting in invalid modelId, this would fail as well. Instead of running and failing I thought skipping would be better.

Reply:
understood